pkg/server/manifest/manifest.go

```diff
@@ -0,0 +1,100 @@
+package manifest
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/tensorleap/leap-cli/pkg/log"
+	"gopkg.in/yaml.v3"
+)
+
+type ExternalImagesInfo struct {
+	K3dToolsImage               string `yaml:"k3dToolsImage"`
+	K3sImageVersion             string `yaml:"k3sImageVersion"`
+	K3sGpuImageVersion          string `yaml:"k3sGpuImageVersion"`
+	K3sGpuImageTagSuffix        string `yaml:"k3sGpuImageTagSuffix"`
+	K3sRegistryUrl              string `yaml:"k3sRegistryUrl"`
+	RegisterImage               string `yaml:"registerImage"`
+	HelmRepoUrl                 string `yaml:"helmRepoUrl"`
+	ServerChartName             string `yaml:"serverChartName"`
+	ServerReleaseName           string `yaml:"serverReleaseName"`
+	CheckDockerRequirementImage string `yaml:"checkDockerRequirementImage"`
+}
+
+type HelmChartMeta struct {
+	Version     string `yaml:"version"`
+	RepoUrl     string `yaml:"repoUrl"`
+	ChartName   string `yaml:"chartName"`
+	ReleaseName string `yaml:"releaseName"`
+}
+
+type InstallationManifest struct {
+	ManifestVersion             string        `yaml:"manifestVersion"`
+	RegisterImage               string        `yaml:"registerImage"`
+	K3dToolsImage               string        `yaml:"k3dToolsImage"`
+	K3sImage                    string        `yaml:"k3sImage"`
+	K3sImages                   []string      `yaml:"k3sImages"`
```

Comment:
Another good reason to test the GPU on non GPU devices
And also to try to push all the images to our GCR